packages/system/foundationdb-operator/Makefile

```
@@ -0,0 +1,19 @@
+export NAME=foundationdb-operator
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)